src/pages/saksoversikt/vilkår/PersonligOppmøte.tsx

```diff
@@ -54,12 +56,19 @@ const PersonligOppmøte = (props: VilkårsvurderingBaseProps) => {
         );
     };
 
-    const onInit = (personligOppmøte: PersonligOppmøteType | null): FormData => {
+    const onInit = (personligOppmøte: Nullable<PersonligOppmøteType>, fraSøknad: Nullable<Vergemål>): FormData => {
         if (!personligOppmøte) {
+            if (!fraSøknad) {
+                return {
+                    status: MøttPersonlig.Ja,
```

Comment:
Hvis vi ikke har personligOppmøte, og heller ikke noe fra søknad, hvorfor defaulter vi til 'MøttPersonlig.ja' ? Burde den heller ikke bli null som default?

Reply:
Hmm, jeg tolket dataen fra søknaden som at dersom `harFullmektigEllerVerge` er `null` så betyr det at søker møtte personlig.
Kanskje navngivingen her er dum; feltet fra søknaden heter jo `harFullmektigEllerVerge`

Reply:
ah, ja riktig. tolket personligOppmøte mer som brukeren selv, enn det feltet fra søknaden. 👍

Reply:
Hehe, ja. Oppdaterte navnet på variablene for å prøve å gjøre det litt tydeligere.